src/pkg/fault/item.go

```diff
@@ -0,0 +1,79 @@
+package fault
+
+type itemKind string
```

Comment:
I understand that we already have `itemType`. However, it does seem to work in this connotation. There are other things that may be more useful:
- class
- division
- group

Reply:
* `class` too overloaded a term in software
* `division` makes me think that this is a divider, which it is not.
* `group` could work, but imo it reads identically to `kind`.